src/www/status_interfaces.php

```diff
@@ -338,6 +338,14 @@ interface_configure(false, $interface, true);
                         endforeach;?>
                       </td>
                     </tr>
+<?php
+                    endif;
+                    if (file_exists("/tmp/{$ifinfo['if']}_pdinfo")): ?>                    
```

Comment:
in the long run it might be better to pull this through interfaces statistics provider functions, but for now this will suffice except that ifinfo['if'] is probably the wrong interface in some edge cases of IPv6